woocommerce-connect-client.php

```diff
@@ -101,6 +120,14 @@ public function set_service_settings_store( WC_Connect_Service_Settings_Store $s
 			$this->service_settings_store = $settings_store;
 		}
 
+		public function get_tracks() {
```

Comment:
What I find odd about all of this is that you're only using tracks via the static methods above. What's the point of initializing an instance of this class if it's not being used anywhere?